content/ADR-175-remove-assets-from-build-local-addressables.md

```diff
@@ -0,0 +1,128 @@
+   - Assets packed together/separately according to the required loading way: any combination of "Include Addresses in Catalog", "Include GUIDs in Catalog" and "Include Labels in Catalog". Thus, we will reduce the size of Catalog.
+   - Bundle Mode "Pack Together by Label" will be used to differentiate between platforms (WebGL, Standalone) within the same logical group to exclude unused assets (and the related overhead) if needed
+   - We may come up with additional rules for our convenience
+
+Check [Unity guidelines](https://docs.unity3d.com/Packages/com.unity.addressables@1.21/manual/AddressableAssetsDevelopmentCycle.html#organizing-addressable-assets) for additional information.
+
+#### Asset Bundle CRC	
+At the moment we don't have any sensitive data that requires integrity check. Moreover, for bigger files it will cause noticeable overhead on loading.
+So for the groups we will disable it.
+
+### Resources handling
+We are going to exclude Scenes and Resources from Addressables Catalog:
+   - We don't plan to migrate "Resources"' API to the corresponding one of "Addressables"
+   - We plan to migrate assets currently added to "Resources" to "Addressables" with a proper distribution over time
+
+<figure>
+  <img src="/resources/ADR-175/ResourceHandling.png" />
+</figure>
+
+### Addressables Caching
+"Addressables" uses right the same "Caching" system that "Asset Bundles" does. It works out of the box for all the supported platforms. It is possible to disable caching per group, but in our case we will not be doing it.
+
+### Asset Reference
+We are going to utilize ["AssetReference"](https://docs.unity3d.com/Packages/com.unity.addressables@1.21/manual/AssetReferences.html) and "AssetLabelReference" instead of "SerializeField" to properly reference Assets that should be a part of a separate Addressable without creating a strong reference to them.
+
+We can think of several use cases:
+   - Optional references that are loaded on demand in conditional branches, and, thus, should not be present in memory all the time
+   - Avoiding the case of referencing the same assets by strong references (that are included in the build) and including them in the Addressable/Asset Bundle in order to prevent duplication
+
+Real use cases and distribution of such assets in groups will be refined and defined according to the feature production pipeline with performance in mind.
+
+### Addressables Disposal
+Currently, we don't have a robust mechanism for cleaning up memory from assets. We are not going to introduce it this time either. Thus, we will keep all the Addressables loaded in memory. Considering it's the "Local" mode only, it's not gonna be a problem.
+
+### Simulation
+We will use "Simulate Groups" mode for loading Addressables in the Editor. It will help us detect and investigate problems earlier. We will prefer it over "Asset Database" false mode.
```

Comment:
```suggestion
We will use the "Simulate Groups" mode for loading Addressables in the Editor. It will help us detect and investigate problems earlier. We will prefer it over the "Asset Database" false mode.
```